core/lib/spree/core/controller_helpers/order.rb

```diff
@@ -37,6 +37,12 @@ def current_order(options = {})
         end
 
         def associate_user
+          return if Spree::Config.associate_user_in_authentication_extension
+          Spree::Deprecation.behavior = :stderr
```

Comment:
Do we want to override whatever preferences users have?

Reply:
@waiting-for-dev  by default Spree::Deprecation has behavior :warn and return ActiveSupport::DeprecationException  and integration tests capybara are fails. By this reason I've overridden behavior to stderr.

Reply:
I see your point. However, as we're using `#by_version` in the configuration class now, the master branch will use `true` (as its version is `3.2.0.alpha`), and there's no actual need for it anymore.